test/ImageminPlugin.test.js

```diff
@@ -1358,7 +1358,7 @@ describe("imagemin plugin", () => {
     expect(warnings).toHaveLength(0);
     expect(errors).toHaveLength(1);
     expect(errors[0].message).toMatch(
-      /Multiple values for the 'encodeOptions' option is not supported for 'loader-test.png', specify only one codec for the generator/
+      /Multiple values for the 'encodeOptions' option is not supported for '.+loader-test.png', specify only one codec for the generator/
```

Comment:
Why it was changed?

Reply:
@alexander-akait After fixing `isAbsolute` function `original.filename` in implementations becoms a relative path and doesn't match previous regexp.